sklearn/ensemble/tests/test_forest.py

```diff
@@ -197,9 +197,6 @@ def check_importances(name, X, y):
         assert_equal(importances.shape[0], 10)
         assert_equal(n_important, 3)
 
-        X_new = clf.transform(X, threshold="mean")
```

Comment:
I would rather catch the deprecation warning and remove the test when we remove the function.